code/controllers/subsystem/vote.dm

```diff
@@ -274,12 +299,18 @@ SUBSYSTEM_DEF(vote)
 		if("toggle_gamemode")
 			if(usr.client.holder)
 				CONFIG_SET(flag/allow_vote_mode, !CONFIG_GET(flag/allow_vote_mode))
+		if("toggle_map")
+			if(usr.client.holder)

```

Comment:
These should be same as the interface checks but i guess that's for another PR.

Reply:
I could fix that in this PR if you want

Reply:
ill just add the check